packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -85,7 +85,7 @@ export const BookingFields = ({
 
         // Attendee location field can be edited during reschedule
         if (field.name === SystemField.Enum.location) {
-          if (locationResponse?.value === "attendeeInPerson" || "phone") {
+          if (locationResponse?.value === "attendeeInPerson" || "phone" || "somewhereElse") {
```

Comment:
I've just noticed something - that is in this PR and also main 

locationResponse?.value === "attendeeInPerson" || "phone" || "somewhereElse"

If `locationResponse?.value === "attendeeInPerson" ` is false we goto "phone"

Which in theory returns true even when we might not want to 

`Boolean("phone") === true` I think we need to do something simular to the files above  `apps/web/components/dialog/EditLocationDialog.tsx` to check if it is in the array

Reply:
Yeah, and also let's add a test to confirm this works (obviously this is entirely uncovered or otherwise it'd be exposed earlier)